addons/website/views/snippets/s_split_intro.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_split_intro" name="Split Intro">
+    <section class="s_split_intro">
+        <div class="container-fluid">
+            <div class="row s_nb_column_fixed">
+                <div class="col-lg-6 order-lg-0 pt40 pb40 px-5 o_cc o_cc1" style="order: 1;">
+                    <h1 class="display-3">Revolutionize your<br/>business operations</h1>
+                    <p class="lead"><br/>Step into the future with our innovative solutions tailored to meet the unique needs of your business. From improving efficiency to driving growth, our tools provide the foundation for sustainable success. Don’t let outdated processes hold you back any longer.<br/><br/></p>
+                    <a t-att-href="cta_btn_href" class="btn btn-lg btn-primary"><t t-out="cta_btn_text">Discover more</t></a>
+                </div>
+                <div class="o_not_editable o_colored_level o_cc o_cc4 pt80 pb80 oe_img_bg o_bg_img_center col-lg-6 order-lg-0" style="background-image: url('/web/image/website.s_split_intro_default_image'); order: 0;"/>
```

Comment:
```suggestion
                <div class="oe_img_bg o_bg_img_center pt120 pb120 col-lg-6 offset-lg-1 order-lg-0" style="background-image: url('/web/image/website.s_split_intro_default_image'); order: 0;"/>
```

- IMO this image should be editable
- `o_cc o_cc4` don't really look necessary + there is an visual glitch in the modal if we keep them:

| With o_cc | Without o_cc |
|--------|--------|
| ![Screenshot 2024-09-11 at 10 35 24](https://github.com/user-attachments/assets/c8df7fc3-3206-439a-ac64-ac4fb57c1c74) | ![Screenshot 2024-09-11 at 10 35 32](https://github.com/user-attachments/assets/d8d87dd0-1c1f-4f67-9e83-d29f6a52dfe3) |